server/graphql/v2/query/CryptoExchangeRateQuery.js

```diff
@@ -0,0 +1,22 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+
+import { getCryptoFxRate } from '../../../lib/currency';
+
+const CryptoExchangeRateQuery = {
+  type: GraphQLString,
+  args: {
+    cryptoCurrency: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'Crypto currency symbol. Example: BTC',
+    },
+    collectiveCurrency: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'Fiat currency symbol. Example: USD',
+    },
+  },
+  async resolve(_, args) {
+    return getCryptoFxRate(args.cryptoCurrency, args.collectiveCurrency);
```

Comment:
Why do we need something specific to crypto? Could we consider something generic instead `exchangeRateQuery `

Reply:
It's overly specific, if it's in GraphQL API v2, I want something semantically clean and generic.

Reply:
Ah so you mean we change this query should support both crypto exchange rates and non-crypto exchange rates (user can pass in crypto currency or fiat currency symbols)? Or do you mean we just rename it as `exchangeRateQuery`? The part I didn't quite get is renaming this to `exchnageRateQuery` doesn't tally with what it does. Currently it only returns the exchange rate for crypto currencies. 🤔

Reply:
> Ah so you mean we change this query should support both crypto exchange rates and non-crypto exchange rates (user can pass in crypto currency or fiat currency symbols)?

Yes